dxr/templates/nunjucks/promoted.html

```diff
@@ -0,0 +1,27 @@
+{% macro promoted_list(query, promoted, promoted_count, www_root, tree) -%}
+  {% for result in promoted %}
+    <div class="result-head">
+      <div class="left-column">
+        <div class="icon-container {{ result.iconClass }}"></div>
+      </div>
+      <div>
+        {{ result.pathLine | safe }}
+      </div>
+    </div>
+  {% endfor %}
+{%- endmacro %}
+
+{%- macro promoted_header(promoted_query, promoted_count, promoted_count_formatted, top_of_tree, tree) -%}
+  <p class="promoted-see-more">
+    {% if promoted_count == 1 %}
+      Found 1 path result:
+    {% elif promoted_count <= 5 %}
+      Showing all of <a href="{{ promoted_query }}">{{ promoted_count_formatted }} path results</a>:
```

Comment:
I think we can just say "2 path results" here, and I don't think we need the link. I can't think of a common case in which someone would need the query link when there are very few results, can you?